target/linux/ath79/image/generic.mk

```diff
@@ -1213,11 +1213,11 @@ define Device/dlink_dir-825-b1
   DEVICE_PACKAGES := kmod-usb-ohci kmod-usb2 kmod-usb-ledtrig-usbport \
 	kmod-leds-reset kmod-owl-loader kmod-switch-rtl8366s
   IMAGE_SIZE := 7808k
-  FACTORY_SIZE := 6144k
-  IMAGES += factory.bin
-  IMAGE/factory.bin = append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | \
-	pad-rootfs | check-size $$$$(FACTORY_SIZE) | pad-to $$$$(FACTORY_SIZE) | \
-	append-string 01AP94-AR7161-RT-080619-00
+# FACTORY_SIZE := 6144k
+# IMAGES += factory.bin
```

Comment:
I think it is sufficient to comment this line only.

Reply:
Fixed, thanks.

The buildbot failed to build `ath79/generic` yesterday. Hope this PR can fix it. Ref: https://buildbot.openwrt.org/images/#/builders/219/builds/251